medication statements/README.md

```diff
@@ -0,0 +1,363 @@
+    "dosage": [
+      {
+        "text": "To Be Used As Directed"
+      }
+    ]
+  }
+}
+```
+</details>
+
+## JSON FHIR > XML HL7
+
+A XML HL7 `MedicationStatement` is mapped from a FHIR `medicationRequest`.  This can be one of two types, `ORDER` or `PLAN` which is determined by value of `medicationRequest.intent`
+
+### Order
+
+For details of order mapping see the JSON FHIR > XML HL7 mapping for [Medication](../medications/README.md)
+
+### Plan
+
+| Mapped to (XML HL7)                                                                                                | Mapped from (JSON FHIR / other source )                                                                                            |
+|--------------------------------------------------------------------------------------------------------------------|------------------------------------------------------------------------------------------------------------------------------------|
+| MedicationStatement \[@classCode] \[@moodCode]                                                                     | fixed value `@classCode="SBADM"`, fixed value `@moodCode="INT"`                                                                    |
+| MedicationStatement / id \[@root]                                                                                  | adaptor generated UUID                                                                                                             |
+| MedicationStatement / statusCode                                                                                   | `medicationRequest.status` when `"active"` = `"ACTIVE"` otherwise is `"COMPLETE"`                                                  |
+| MedicationStatement / effectiveTime / low \[@value]                                                                | `medicationRequest.dispenseRequest.validityPeriod.start`                                                                           |
+| MedicationStatement / effectiveTime / high \[@value]                                                               | `medicationRequest.dispenseRequest.validityPeriod.end` <sup>1</sup>                                                                |
+| MedicationStatement / availabilityTime  \[@value]                                                                  | `medicationRequest.dispenseRequest.validityPeriod.start`                                                                           |
+| MedicationStatement / consumable \[@typeCode] / manufacturedProduct \[@typeCode="CSM"]                             | fixed value = `"CSM"`                                                                                                              |
+| MedicationStatement / consumable / manufacturedProduct \[@typeCode="CSM"]                                          | fixed value = `"MANU"`                                                                                                             |
```

Comment:
Should the `typeCode` here be `classCode` instead?

Reply:
Indeed it should